content/tbb/how-to-verify-signature/contents.lr

```diff
@@ -85,7 +85,13 @@ The result of the command should produce something like this:
     gpgv: Signature made 07/08/19 04:03:49 Pacific Daylight Time
     gpgv:                using RSA key EB774491D9FF06E2
     gpgv: Good signature from "Tor Browser Developers (signing key) <torbrowser@torproject.org>"
-    
+
+#### Public key
```

Comment:
The title of this section should say that it is a workaround; an alternative solution. Some users could think it's an extra step needed.

Reply:
@gusgustavo What do you think of it now?